apps/dashboard/app/(app)/audit/[bucket]/row.tsx

```diff
@@ -85,10 +86,10 @@ export const Row: React.FC<Props> = ({ auditLog, user }) => {
         <TableCell>
           <div className="flex items-center gap-2">
             <span className="text-sm text-content">
-              {new Date(auditLog.time).toLocaleDateString()}
+              {format(new Date(auditLog.time), 'dd/MM/yyyy')}
```

Comment:
I don't think this is a good change
it'll confuse the americans probably

what was the reason behind it?

Reply:
I guess you'r right, this solution is not ideal. The reason is in the commit message:

```bash
Replace `toLocaleDateString()` with `date-fns` to ensure consistent date formatting between server and client.
This prevents hydration mismatches caused by differing time zones or locale settings, ensuring stable rendering across environments.
```

I'm thinking now we should extract this to separate component that will have `use client` header and use `.toLocaleDateString()`. This should solve the hydration error as well as suit anyone worldwide.  WDYT?

Reply:
ah I missed that. Let's use this for now, I've done it in the past and never noticed any issues
https://nextjs.org/docs/messages/react-hydration-error#solution-3-using-suppresshydrationwarning

Reply:
Fixed ✔️